README.md

```diff
@@ -1,37 +1,44 @@
 # Veres One DIDs
 
 This library provides support classes for creating and processing
-Decentralized Identifiers for Veres One. This library enables a developer to:
+Decentralized Identifiers for [Veres One](https://veres.one/). This library
+enables a developer to:
 
 * Create a Veres One DID
-* Perform Veres One key rotation
 * Generate Veres One cryptographic proofs
 
-## The Promises API
-
-  * api.generate(options)
-
-## The Callback API (node.js)
-
-  * api.generate(options, (err, didDocument))
-
-## Quick Examples
-
 ```
 npm install did-veres-one
 ```
 
 ```js
-const didv1 = require('did-veres-one');
-const options = {};
+const v1 = require('did-veres-one');
 
-// generate the DID document
-const didDocument = await didv1.generate();
+// See Configuration below for list of options
+const options = {mode: 'dev', httpsAgent, hostname: 'localhost:12345'};
+const veresDriver = v1.driver(options);
 ```
 
 ## Configuration
 
-For documentation on configuration, see [config.js](./lib/config.js).
+* options - a set of options used when generating the DID Document
+  * didType - the type of DID to generate.
```

Comment:
Be consistent and use backticks on all these rather than just invokeKey.

Reply:
Done, thanks.